### PR TITLE
Added documentation repo link to FAQ

### PR DESCRIPTION
**Pull request by satyaakam**
Added the documentation repo link for FAQ

### CODE REVIEW

File: _posts/2015-04-05-faq.md [hunk 1]
```diff
@@ -203,7 +203,9 @@ Some bugs are more important than others.  Stability and usability issues are ve
...
+Don't worry if you are not a proficient developer -- there are many ways to help out. Documentation is good way to start contributing you can start here [Documentation](https://github.com/bigbluebutton/bigbluebutton.github.io), You can also become proficient in the installation and configuration of a BigBlueButton server.  Each month, there are many new users in [bigbluebutton-setup](https://groups.google.com/forum/#!forum/bigbluebutton-setup) that need help with setup of BigBlueButton, especially setup behind a [firewall](#can-i-provide-external-access-to-a-bigbluebutton-server-behind-my-firewall).    You can [contribute a language file](/dev/localization.html). You could point out any errors to this documentation.  Such assistance reduces the support load on us and gives us more time to work on improving BigBlueButton.
```
Comment (matiasilva) [inline _posts/2015-04-05-faq.md:206]:
```suggestion
Don't worry if you are not a proficient developer -- there are many ways to help out. Documentation is an easy and helpful way to start contributing.  You can follow the instructions on getting started [here](https://github.com/bigbluebutton/bigbluebutton.github.io). You can also become proficient in the installation and configuration of a BigBlueButton server.  Each month, there are many new users in [bigbluebutton-setup](https://groups.google.com/forum/#!forum/bigbluebutton-setup) that need help with setup of BigBlueButton, especially setup behind a [firewall](#can-i-provide-external-access-to-a-bigbluebutton-server-behind-my-firewall).    You can [contribute a language file](/dev/localization.html). You could point out any errors to this documentation.  Such assistance reduces the support load on us and gives us more time to work on improving BigBlueButton.
```

**Comment by matiasilva** [comment]
This is a simple and helpful PR. I've just tweaked a few things to make it more inviting. Hope this can be merged ASAP.

File: _posts/2015-04-05-faq.md (hunk 1 above)
Comment (satyaakam) [reply _posts/2015-04-05-faq.md:206]: Yes sounds much better :-)

Comment (basisbit) [reply _posts/2015-04-05-faq.md:206]: @satyaakam if you agree to this change, please adjust your pull request by changing the file in your fork (satyaakam:FAQ-change) to automatically add that commit to this pull request.

**Comment by antobinary** [comment]
I tried to merge this but the context has changed quite a bit since. Thank you for your contribution still! Looks like the current version shares the message of what you wanted conveyed... :wink:

**Comment by satyaakam** [comment]
haaah i lost the track of this let me check on it in the weekend

**Pull request closed without merging**